add a couple more bugs to historical updates page (#566)
PureStake/moonbeam-docs: builders/build/historical-updates.md

***

#### Ethereum Fees Weren't Sent to Treasury {: #ethereum-fees-to-treasury }

> The fee model on Moonbeam for transactions and smart contract execution are handled so that 20% of the fees go to the on-chain treasury and 80% are burned as a deflationary force. Prior to runtime 800, Ethereum transactions did not result in 20% of the transaction fees going to the on-chain Treasury.

Reviewer: ```suggestion
The fee model on Moonbeam for transactions have 20% of the fees go to the on-chain treasury and 80% burned as a deflationary force. Prior to runtime 800, Ethereum transactions did not result in 20% of the transaction fees going to the on-chain Treasury.
```
